README.md

```diff
@@ -260,25 +151,10 @@ If you like this project...
 
 > Give credit to everyone who inspired your codebase.
 
```

Comment:
- Hi :clap:, Since you've not deployed the project, kindly remove the `live demo` and `Deployment` sections from the top navigation and from the readme body to make the readme more professional.